Solutions/VMware vCenter/Analytic Rules/vCenter-Root impersonation.yaml

```diff
@@ -0,0 +1,29 @@
+id: f1fcb22c-b459-42f2-a7ee-7276b5f1309c
+name: vCenter - Root impersonation
+description: |
+  'Detects when root impersonation occurs.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: vCenter
+    dataTypes:
+      - vCenter
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1078
+query: |
+ vCenter
+  | where Message has_all ('ImpersonateUser', 'VcIntegrity', 'root')
+  | extend user = 'root'
+  | extend AccountCustomEntity = user
```

Comment:
Do not use custom entity mapping, it is legacy and can cause issues with the rules wizard.